Выполнил домашнюю работу 2

NikGusakov opened the pull request:
- Создал метод для проверки счастливых билетов
- Создал тесты для этого метода
- Создал сьюты для негативных, позитивных и всех тестов
- Настроил запуск тестов с помощью maven surefire
- Настроил профили maven для запуска тестов

NikGusakov committed:
Выполнил домашнюю работу 2
- Создал метод для проверки счастливых билетов
- Создал тесты для этого метода
- Создал сьюты для негативных, позитивных и всех тестов
- Настроил запуск тестов с помощью maven surefire
- Настроил профили maven для запуска тестов

khda91 commented on nikita-gusakov-level-up/src/main/java/ru/levelp/at/homework2/HappyTicket.java:
```diff
@@ -0,0 +1,32 @@
+
+        boolean result = false;
+
+        if (number.length() == 6) {
```
А если длина не 6 символов разве это не ошибка?

NikGusakov replied:
Исправил работу метода так, что теперь при получении числа длинной НЕ 6 символов возникает исключение

khda91 reviewed (changes requested):
Необходимо исправить комментарий

NikGusakov committed:
Исправил реализацию согласно замечаниям
- Исправил работу метода
- Исправил отображение ошибки в main()
- Исправил набор тестовых данных

khda91 reviewed (approved):
Работа принята
